insights/parsers/lpstat.py

```diff
@@ -87,3 +89,29 @@ def printer_names_by_status(self, status):
         """
         names = [prntr['name'] for prntr in self.printers if prntr['status'] == status]
         return names
+
+
+@parser(Specs.lpstat_protocol_printers)
+class LpstatProtocol(CommandParser):
+    """
+    Class to parse ``lpstat -v`` command output.
+
+    Sample output of the command::
+        device for NAY_10F_Smurfs: ipp
+        device for PEK_8F_Autumn: ipp
+
+    Examples:
+        >>> type(lpstat_protocol)
+        <class 'insights.parsers.lpstat.LpstatProtocol'>
+        >>> lpstat_protocol['NAY_10F_Smurfs']
+        'ipp'
+    """
+    def parse_content(self, content):
+        self.data = {}
```

Comment:
@wushiqinlou since you are only created a `dict` as the attribute, just go ahead and make the base class `dict` and then you'll get all of the methods for free.  [Here's an example](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/hammer_ping.py) of how you can implement the parser as a `dict`.

Reply:
@bfahr - Updated, please have a check.